libs/runtime/unit/platform_unit_test.cpp

```diff
@@ -1208,11 +1208,11 @@ has_dominant_frequency(size_t sequence_length, std::function<uint32_t()> random_
 
 TEST_CASE("verify random", "[platform]")
 {
-    KIRQL old_irql = KeRaiseIrqlToDpcLevel();
-
     _test_helper test_helper;
     test_helper.initialize();
 
+    KIRQL old_irql = KeRaiseIrqlToDpcLevel();
```

Comment:
Can we make this lower on scope exit?

Reply:
> Can we make this lower on scope exit?

Not sure I follow? There _is_ a `KeLowerIrql` call at the end of that test function, or did I misunderstand your comment?